src/displayapp/screens/WatchFaceFuturistic.cpp

```diff
@@ -0,0 +1,185 @@
+#include "displayapp/screens/NotificationIcon.h"
+#include "components/heartrate/HeartRateController.h"
+#include "displayapp/InfiniTimeTheme.h"
+
+using namespace Pinetime::Applications::Screens;
+
+WatchFaceFuturistic::WatchFaceFuturistic(Pinetime::Applications::DisplayApp* app,
+                                         Controllers::DateTime& dateTimeController,
+                                         Controllers::Battery& batteryController,
+                                         Controllers::Ble& bleController,
+                                         Controllers::NotificationManager& notificationManager,
+                                         Controllers::HeartRateController& heartRateController)
+  : Screen(app),
+    currentDateTime {{}},
+    dateTimeController {dateTimeController},
+    batteryController {batteryController},
+    bleController {bleController},
+    notificationManager {notificationManager},
+    heartRateController {heartRateController} {
+
+  // heart beat icon
+  heartbeatIcon = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(heartbeatIcon, Symbols::heartBeat);
+  lv_obj_set_style_local_text_color(heartbeatIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(outerTextColor));
+  lv_obj_align(heartbeatIcon, lv_scr_act(), LV_ALIGN_IN_BOTTOM_LEFT, 0, 0);
+
+  // heart rate
+  heartbeatValue = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text(heartbeatValue, "000");
+  lv_obj_set_style_local_text_color(heartbeatValue, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(outerTextColor));
+  lv_obj_align(heartbeatValue, heartbeatIcon, LV_ALIGN_OUT_RIGHT_MID, 5, 0);
+
+  // plug icon
+  plugIcon = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(plugIcon, Symbols::plug);
+  lv_obj_set_style_local_text_color(plugIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(outerTextColor));
+  lv_obj_align(plugIcon, nullptr, LV_ALIGN_IN_TOP_LEFT, 0, 0);
+
+  // battery percentage (only displayed when charging)
+  batteryStatus = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text(batteryStatus, "000%");
+  lv_obj_set_style_local_text_color(batteryStatus, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(outerTextColor));
+  lv_obj_align(batteryStatus, plugIcon, LV_ALIGN_OUT_RIGHT_MID, 5, 0);
+
+  // low battery alert message
+  batteryAlert = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text(batteryAlert, "low bat");
+  lv_obj_set_style_local_text_color(batteryAlert, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(outerTextColor));
+  lv_obj_align(batteryAlert, nullptr, LV_ALIGN_IN_BOTTOM_RIGHT, 0, 0);
+
+  sHour = 99;
+  sMinute = 99;
+  sSecond = 99;
+
+  lv_style_init(&arc_fg_style);
+  lv_style_set_line_rounded(&arc_fg_style, LV_STATE_DEFAULT, false);
+  lv_style_set_line_width(&arc_fg_style, LV_STATE_DEFAULT, 8);
+
+  lv_style_init(&arc_bg_style);
+  lv_style_set_line_width(&arc_bg_style, LV_STATE_DEFAULT, 0);
+
```

Comment:
Using styles is usually less efficient than setting local style. Please test the difference in build size.